.github/workflows/docker-latest.yml

```diff
@@ -0,0 +1,48 @@
+name: Publish Docker image for latest release
+
+on:
+  push:
+    tags:
+      - release
+
+env:
+  TEST_TAG: zeek:latest
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Checkout submodules
+        run: git submodule update --init --recursive
+      - uses: docker/setup-qemu-action@v1
```

Comment:
Could you say something here as to why we need this? Is it for the `container-structure-test`?

Reply:
No, this has nothing to do with the tests, also see https://github.com/zeek/zeek/pull/1739#discussion_r713183044.